frontends/web/src/api/accountsync.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { subscribeEndpoint } from './subscribe';
+import { Unsubscribe } from '../utils/event';
+import { getAccounts, IAccount} from './account';
+
+export const accounts = (cb: (acconuts: IAccount[]) => void): Unsubscribe => {
+    const unsubscribe = subscribeEndpoint('accounts', cb);
+    getAccounts().then(accounts => cb(accounts)).catch(console.error);
```

Comment:
Shouldn't this report the error to the user somehow, instead of logging to the console? Here and in other places.

Reply:
yes, every .catch(console.error) should go away, especially the ones in api and handled in the view.

This promise rejection catch is currently missing in our codebase iirc.

Reply:
Passing this promise to view component is a bit tricky as all subscibers should return the unsubscribe function.

Now I am thinking about removing the line completely from here, so the api consumer would have to do `getAccounts()` in the view.